packages/opentelemetry-plugin-grpc/src/grpc-types.ts

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
Can we just say `types.ts` instead of `grpc-types.ts`?

Reply:
Renamed this (and AttributeNames, etc) in 2d01b61